### PR TITLE
update inputdata for Bu angle fix; re-enable cpld C48 warmstart test

### LINKED ISSUE #1822: re-enable c48-5deg warmstart test
## Description

The C48-5deg warmstart test was disabled in PR #1812. It needs to be re-enabled.

## Solution

Once PR #1777 and #1807 are merged, a new set of warm start ICs should be created and added to the input-data directory. This will allow the test to be re-enabled.

### LINKED ISSUE #2228: grid files used by CICE contain incorrect rotation angle
## Description
All applications using CICE use a resolution-dependent fixed file ``grid_cice_NEMS_mx${OCNRES}.nc`` to define the CICE grid. This file contains the vector rotation angle on corners (``Bu``) grid points. This field is incorrect. See Issue https://github.com/ufs-community/UFS_UTILS/issues/921 for a complete description. The current rotation angle at 1/4 deg have a maximum angular error of ~20deg.

A new input-data directory is required to update the required files.

### PR DESCRIPTION
**Pull request by DeniseWorthen**
## Commit Queue Requirements:

- [x] Fill out all sections of this template.
- [ ] All sub component pull requests have been reviewed by their code managers. **NA**
- [x] Run the full Intel+GNU RT suite (compared to current baselines) on either Hera/Derecho/Hercules
- [x] Commit 'test_changes.list' from previous step
---
## Description:

Creates a new input-data directory to account for Bu angle fix for CICE and re-enabling cpld C48 tests. The following files or directories are updated:

```
./CICE_FIX/500/grid_cice_NEMS_mx500.nc
./CICE_FIX/500/kmtu_cice_NEMS_mx500.nc
./CICE_FIX/100/grid_cice_NEMS_mx100.nc
./CICE_FIX/100/kmtu_cice_NEMS_mx100.nc
./CICE_FIX/025/grid_cice_NEMS_mx025.nc
./CICE_FIX/025/kmtu_cice_NEMS_mx025.nc
./CICE_FIX/050/grid_cice_NEMS_mx050.nc
./CICE_FIX/050/kmtu_cice_NEMS_mx050.nc

./CICE_FIX/500/mesh.mx500.nc
./CICE_FIX/100/mesh.mx100.nc
./CICE_FIX/025/mesh.mx025.nc
./CICE_FIX/050/mesh.mx050.nc

./FV3_input_data48/INPUT_L127_mx500/2021032306
./MOM6_IC/500/2021032306
./CICE_IC/500/2021032306
./CMEPS_IC/C48mx500/2021032306
```

The following files or directories are removed

```
./CPL_FIX/aC48o400
./FV3_input_data48/INPUT_L127_mx400
./MOM6_FIX/400
./lake_p8_water_fraction2020/oro_C48.mx400.tile4.nc
./lake_p8_water_fraction2020/oro_C48.mx400.tile5.nc
./lake_p8_water_fraction2020/oro_C48.mx400.tile1.nc
./lake_p8_water_fraction2020/oro_C48.mx400.tile3.nc
./lake_p8_water_fraction2020/oro_C48.mx400.tile6.nc
./lake_p8_water_fraction2020/oro_C48.mx400.tile2.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile4.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile5.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile1.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile3.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile6.nc
./FV3_fix_tiled/C48/oro_C48.mx400.tile2.nc
./CICE_IC/400
./CICE_FIX/400
./CICE_FIX/400/grid_cice_NEMS_mx400.nc
./CICE_FIX/400/kmtu_cice_NEMS_mx400.nc
./CICE_FIX/400/mesh.mx400.nc
```

Note the mesh files are not directly impacted by the Bu angle bug, but during development of the fix, the 360. deg increment added to the MOM6 longitudes was removed. This results in a change of the mesh file longitude coordinate.

The updated files were copied from the UFS-UTILS baseline directory that was created by https://github.com/ufs-community/UFS_UTILS/pull/922. See related issue https://github.com/ufs-community/UFS_UTILS/issues/921 

A new input data directory has been staged on Hera at ``/scratch1/NCEPDEV/stmp4/Denise.Worthen/input-data_20240501`` and will need to be sync'd across all platforms.

This staged input directory also removes the files related to the "400" ocean resolution. These are currently unused, and and simply copies of the "500" directories. This is because when the files were originally made, the wrong name string was used. We have never had a 4-deg ocean resolution.

It also updates the warmstart files required to re-enable the C48-5deg warmstart and restart from warmstart RTs.

### Commit Message:

```
* UFSWM - update input data directory to fix CICE grid files for incorrect Bu angle, update C48 coupled warmstart files and remove 4-deg ocean files
```

### Priority:

* Normal. 

### UFSWM:

* Closes #2228 
* Closes #1822 

### Sub component Pull Requests:

* None

### UFSWM Blocking Dependencies:

* None

---
### Regression Test Changes (Please commit test_changes.list):

* PR Updates/Changes Baselines.

### Input data Changes:

* Updated input data.

### Library Changes/Upgrades:

* No Updates
  
---

## Testing Log:
- RDHPCS
  - [x] Hera
  - [x] Orion
  - [x] Hercules
  - [x] Jet
  - [x] Gaea
  - [x] Derecho
- WCOSS2
  - [x] Dogwood/Cactus
  - [ ] Acorn
- [x] CI
- [ ] opnReqTest (complete task if unnecessary)

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
@jkbk2004 Why does the new ``input-data-20240501`` on hera still contain files for "400" resolution? If the directory had been sync'd directly from my staged input, there would be no ``400`` files present.

**Comment by jkbk2004** [comment]
> @jkbk2004 Why does the new `input-data-20240501` on hera still contain files for "400" resolution? If the directory had been sync'd directly from my staged input, there would be no `400` files present.

Check now: /scratch2/NAGAPE/epic/UFS-WM_RT/NEMSfv3gfs/input-data-20240501

**Comment by DeniseWorthen** [comment]
These files are still present: ./lake_p8_water_fraction2020/oro_C48.mx400.tile[1-4].nc

Did you rsync directly from my staged input-data. That is the requirement.

**Comment by jkbk2004** [comment]
> These files are still present: ./lake_p8_water_fraction2020/oro_C48.mx400.tile[1-4].nc
> 
> Did you rsync directly from my staged input-data. That is the requirement.

I need sysn admins hep to remove those lake oro (400) files. All others were resynced ok.

**Comment by jkbk2004** [comment]
> > These files are still present: ./lake_p8_water_fraction2020/oro_C48.mx400.tile[1-4].nc
> > Did you rsync directly from my staged input-data. That is the requirement.
> 
> I need sysn admins help to remove those lake oro (400) files. All others were resynced ok.

**Comment by jkbk2004** [comment]
all good now: /scratch2/NAGAPE/epic/UFS-WM_RT/NEMSfv3gfs/input-data-20240501

**Comment by FernandoAndrade-NOAA** [comment]
Gaea and Jet rsync looks good to go.

**Comment by zach1221** [comment]
> Gaea and Jet rsync looks good to go.

Derecho, orion and hercules should be good as well. @jkbk2004 @BrianCurtis-NOAA I think we'll move to start working this PR since we're skipping Jun's for now.

**Comment by zach1221** [comment]
@DeniseWorthen are you able to update the INPUTDATA_ROOT date to 20240501, or are you ok with me making the change?

**Comment by DeniseWorthen** [comment]
@zach1221 Please update it. Thanks.

**Comment by FernandoAndrade-NOAA** [comment]
Leaving a note that Hera timed out on the `regional_atmaq_debug_intel` case, rerunning without limit increase first. Gaea seems to have stopped rt.sh during the tracking of remaining tasks stage, rerunning that as well.

File: tests/rt.sh [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ if [[ "${CREATE_BASELINE}" == false ]] ; then
   fi
 fi
 
-INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data_20240501}
+INPUTDATA_ROOT=${INPUTDATA_ROOT:-${DISKNM}/NEMSfv3gfs/input-data_20240501}
```
Comment (BrianCurtis-NOAA) [inline tests/rt.sh]: Should this be input-data-20240501 and not input-data_20240501 ?

Comment (BrianCurtis-NOAA) [reply tests/rt.sh]: OK nevermind. I had an old hash up on my browser, looks like this was fixed.

**Comment by BrianCurtis-NOAA** [comment]
Acorn will not be ready in a reasonable time, please skip.

**Comment by zach1221** [comment]
@BrianCurtis-NOAA @jkbk2004 we are ready for final review.

**Review by jkbk2004 — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Pull request merged**